src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -1995,4 +1999,55 @@ describe('OnlineGameWrapperComponent of Quarto:', () => {
         }));
     });
 
+
+    describe('view config button', () => {
+
+        it('should show config when clicking on "view config" (default config)', fakeAsync(async() => {
+            // Given a game with default config
+            await prepareTestUtilsFor(UserMocks.CREATOR_AUTH_USER);
+            // When clicking on "view config" button
+            await testUtils.clickElement('#show-config');
+            // Then it should rules config, with the default config selected
+            testUtils.expectElementToExist('#rules-config-component');
+            const selectedConfig: DebugElement = testUtils.findElement('#rules-config-component option:checked');
+            const selectedConfigName: string = selectedConfig.nativeElement.value;
+            expect(selectedConfigName).toBe('Quarto');
+
+            tick(wrapper.configRoom.maximalMoveDuration * 1000);
+        }));
+
+        it('should hide config when clicking on close button', fakeAsync(async() => {
+            // Given a game with config shown
+            await prepareTestUtilsFor(UserMocks.CREATOR_AUTH_USER);
+            await testUtils.clickElement('#show-config');
+            testUtils.expectElementToExist('#rules-config-component');
+            // When clicking on "view config" button
+            await testUtils.clickElement('#close-config');
+            // Then it should close rules config
+            testUtils.expectElementNotToExist('#rules-config-component');
+
+            tick(wrapper.configRoom.maximalMoveDuration * 1000);
```

Comment:
// Given a game with config shown
// When clicking on "view config" button
// Then it should close rules config
QLEBEQ